lib/block_view.go

```diff
@@ -520,6 +532,29 @@ func (bav *UtxoView) CopyUtxoView() (*UtxoView, error) {
 		newView.TransactorNonceMapKeyToTransactorNonceEntry[entryKey] = &newEntry
 	}
 
+	// Copy the LockedBalanceEntries
+	newView.LockedBalanceEntryKeyToLockedBalanceEntry = make(map[LockedBalanceEntryKey]*LockedBalanceEntry,
```

Comment:
I think you properly copied the keys and values. Just make extra sure you don't have this bug in this code:
* https://go.dev/blog/loopvar-preview